.gitignore

```diff
@@ -22,6 +22,7 @@
 *.swp
 *.gcno
 *.gcda
+*.pyc
```

Comment:
You'll want to add `*.pyo` to ignore list, since Python 2 creates those.

Reply:
Done